app/controllers/items_controller.rb

```diff
@@ -18,6 +21,23 @@ def create
     end
   end
 
+  def destroy
+    @item = current_user.items.find(params[:id])
+
+    if @item.destroy
+      flash[:notice] = "Item was deleted successfully."
+    else
+      flash[:alert] = "Item couldn't be deleted. Try again."
+    end
+
+    respond_to do |format|
+      # format.html { render :partial => 'item' }
+      format.js { render 'destroy' }
+    end
+  end
+
+  private
+
   def item_params
     params.require(:item).permit(:name, :user)
```

Comment:
@naimaseales `item_params` only needs to permit the attributes that are submitted by the new item form. 

`:name` is one of the submitted attributes, `:user` is *not* one of the submitted attributes. As `:user` is not a submitted attribute, its not being used and can be removed from line 42:

```diff
-      params.require(:item).permit(:name, :user)
+      params.require(:item).permit(:name)
```